### PR TITLE
Lesson10

### PR DESCRIPTION
**Pull request by Goshozavr**

### CODE REVIEW

File: home_task7/file3.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        }
+        this._format = format
+    }
+    static fromBook(book, fileFormat){
+        return new EBook(book.title, book._author, book._year, fileFormat)
+    }
+}
+
+const newBook = new EBook("Flowers for Algernon", "Daniel Keyes", 1966, "fb2")
+console.log(newBook._title, newBook._author, newBook._year, newBook._format)
```
Comment (sttaran) [inline home_task7/file3.js:88]: use getters

Comment (sttaran) [reply home_task7/file3.js:88]: do not use protected properties outside of class

File: home_task7/file3.js
Comment (sttaran) [inline home_task7/file3.js:1]: you should have 3 files

1. Book.js - contains Book class
2. EBook.js - contains EBook class
3.  file where you import Book and EBook and use them. Do not create these classes in each file

**Review by sttaran — CHANGES_REQUESTED** [review]

File: home_task7/EBook.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    }
+    set format(format){
+        if (format = "png"){
+            console.log("It`s a picture format")
+            return
+        }
+        this._format = format
+    }
+    static fromBook(book, fileFormat){
+        return new EBook(book.title, book._author, book._year, fileFormat)
```
Comment (sttaran) [inline home_task7/EBook.js]: do not use protected properties outside a class, use getters

**Review by sttaran — CHANGES_REQUESTED** [review]

File: home_task7/EBook.js (hunk 2 above)
Comment (Goshozavr) [reply home_task7/EBook.js]: done

**Pull request merged**